it/controllers/WhatYouOweControllerISpec.scala

```diff
@@ -46,6 +48,26 @@ class WhatYouOweControllerISpec extends ComponentSpecBase {
   )(FakeRequest())
 
   val testTaxYear: Int = getCurrentTaxYearEnd.getYear - 1
+  val testDate: LocalDate = LocalDate.parse("2022-01-01")
```

Comment:
this is fine, but ideally its better to use https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html
LocalDate.of 
reason: that the approach used above we using DateFormatting behind the scene and it not guaranteed that exact format YYYY-MM-DD will be used or for example YYYY-DD-YY; but I think should be fine in this instance